openmc/material.py

```diff
@@ -825,9 +825,15 @@ def get_nuclide_densities(self):
 
         return nuclides
 
-    def get_nuclide_atom_densities(self):
-        """Returns all nuclides in the material and their atomic densities in
-        units of atom/b-cm
+    def get_nuclide_atom_densities(self, nuclide: Optional[str] = None):
+        """Returns one or all nuclides in the material and their atomic
+        densities in units of atom/b-cm
+
+        Parameters
+        ----------
+        nuclides : str, optional
+            Nuclide for which atom density is desired. If not specified, the
+            atom density for the all the material is given.
```

Comment:
This parameters section needs to be below the `.. versionchanged:: 0.13.1` directive